draft-ietf-quic-transport.md

```diff
@@ -2358,17 +2419,56 @@ Application Error Code:
   {{app-error-codes}}).
 
 
+## PATH_CHALLENGE Frame {#frame-path-challenge}
+
+Endpoints can use PATH_CHALLENGE frames (type=0x0e) to check reachability to the
+peer, to verify a new path's PMTU, and to perform address validation during
+connection migration.
```

Comment:
Is PATH_CHALLENGE useful for PMTU validation once the address is known to be good?  Why couldn't you simply rely on ACKs to tell you whether the packet got through or not?